v5/ooapi.v5.core/Models/Pagination.cs

```diff
@@ -6,18 +7,14 @@ namespace ooapi.v5.Models;
 
 public class Pagination<T> : ModelBase
 {
-
     public Pagination()
```

Comment:
Ik zou de constructor private maken, en een static `CreateAsync()`. Dan ontstaat er geen onduidelijkheid dat je altijd `LoadData` (die trouwens async suffix mist) moet aanroepen.

Reply:
Samen met Marthijn gekeken en we hebben de Pagination iets netter neergezet, ik ben de unit tests aan het aanpassen op dit moment.

Reply:
Aanpassing aan Pagination doorgevoerd.